.github/workflows/cpp_extra.yml

```
@@ -330,12 +332,22 @@ jobs:
           cd cpp/examples/minimal_build
           ../minimal_build.build/arrow-example
 
+  odbc:
+    needs: check-labels
+    if: >-
+      needs.check-labels.outputs.force == 'true' ||
+      contains(fromJSON(needs.check-labels.outputs.ci-extra-labels || '[]'), 'CI: Extra') ||
+      contains(fromJSON(needs.check-labels.outputs.ci-extra-labels || '[]'), 'CI: Extra: C++')
+    name: C++ ODBC
```

We can omit "C++" in this file because workflow name already includes "C++":

```suggestion
    name: ODBC
```